common/common-ee.h

```diff
@@ -17,4 +17,6 @@
 void schedf(const char *format, ...);
 void flushschedf();
 
+int getThreadPriority(int);
```

Comment:
I'd prefer this had `int threadId` to ensure it's clear (in IDEs or by browsing) what the argument is used for.

-[Unknown]